src/lib/ai/mental-llama/tests/evaluation-system.test.ts

```
@@ -0,0 +1,227 @@
+      expect(quality).toBeDefined()
+      expect(quality.fluency).toBeGreaterThan(0)
+      expect(quality.completeness).toBeGreaterThan(0)
+      expect(quality.reliability).toBeGreaterThan(0)
+      expect(quality.overall).toBeGreaterThan(0)
+      expect(quality.bartScore).toBeDefined()
+    })
+  })
+})
+
+// Export a placeholder function to avoid type errors
+export const exportToAvoidTypeError = true
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)